tests/by-util/test_cut.rs

```diff
@@ -329,3 +329,39 @@ fn test_8bit_non_utf8_delimiter() {
         .succeeds()
         .stdout_check(|out| out == "b_c\n".as_bytes());
 }
+
+#[test]
+fn test_newline_preservation_with_f1_option() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.write("1", "a\nb");
+    let expected = "a\nb\n";
+    ucmd.args(&["-f1-", "1"]).succeeds().stdout_is(expected);
+}
+
+#[test]
+fn test_cut_handles_empty_lines() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.write("1", "a\nb");
+    at.write("2", "a\nb\n");
+    ucmd.args(&["-f1-", "1"]).succeeds().stdout_is_fixture("2");
+}
```

Comment:
This test does the same as `test_newline_preservation_with_f1_option` and so I think it can either be removed or adapted to do something with empty lines as the function name indicates.